src/compass/schemas/s1_cslc_radar.yaml

```diff
@@ -78,6 +78,8 @@ rdr2geo_options:
    compute_local_incidence_angle: bool(required=False)
    # Enable/disable azimuth angle output
    compute_azimuth_angle: bool(required=False)
+   compute_ground_to_sat_east: bool(required=False)
+   compute_ground_to_sat_north: bool(required=False)
```

Comment:
Could you add the corresponding entries in the `defaults` YAML as well?

Reply:
No problem, I added the options in the defaults